tests/TestCase.php

```diff
@@ -58,18 +58,4 @@ protected function getEnvironmentSetUp($app) {
       'prefix'   => '',
     ));
   }
-
-  /**
-   * Get package providers.
-   *
-   * @param  \Illuminate\Foundation\Application  $app
-   *
-   * @return array
-   */
-  protected function getPackageProviders($app)
-  {
-    return [
-      \Orchestra\Database\ConsoleServiceProvider::class,
```

Comment:
Starting after version 6 this stops being its own package and provider. The whole repo is readonly now. This is part of why the expedient path through this fix was to drop support for Laravel below 9.